README.md

```diff
@@ -1 +1,16 @@
-# rails-forms
\ No newline at end of file
+# README
+
+Created by Microverse students:
+
+[Anthony Tapia](https://github.com/AnthonyTC89) 
+[Matheus Silva](https://github.com/matheus-fls) 
+
+as part of the Ruby on Rails Curriculum.
+
+Instruccions:
```

Comment:
Typo in here, should be "Instructions". If it would've been Spanish, then it is "Instrucciones". It's in English though! [`OPTIONAL`]